README.md

```diff
@@ -1,6 +1,108 @@
+```JavaScript
+const tns = require("nativescript");
+const fs = require("fs");
+const path = require("path");
+const packageJsonContent = JSON.parse(fs.readFileSync("./package.json", "utf8").toString());
+
+const projectSettings = {
+	projectDir: process.cwd(),
+	projectId: packageJsonContent.nativescript.id,
+	projectName: path.dirname(process.cwd()),
+	nativeScriptData: packageJsonContent.nativescript
+};
+
+const androidReleaseConfigurationData = {
+	pathToCertificate: "./myCertificate.p12",
+	certificatePassword: "123456"
+};
+
+const platform = "android";
+const buildConfiguration = "release";
+
+tns.cloudBuildService
+	.build(projectSettings, platform, buildConfiguration, androidReleaseConfigurationData)
+	.then(buildResult => console.log(buildResult))
+	.catch(err => console.error(err));
+```
+
+* `validateBuildProperties` - validates all properties required for specific platform. This includes check if current application identifier matches the CodeSigning identity for iOS operations, check if the specified device identifier (in case there's such) is included in the mobile provision, validation of the password, etc.
+</br>
+Definition:
+
+```TypeScript
+/**
+ * Validates the build properties for specified platform (Android or iOS).
+ * The result promise is rejected with the error found. In case everything is correct, the promise is resolved.
+ * @param {string} platform The mobile platform for which the application should be built: Android or iOS.
+ * @param {string} buildConfiguration The build configuration - Debug or Release.
+ * @param {string} projectId Application identifier of the project.
+ * @param {IAndroidBuildData} androidBuildData Android speicific information for the build.
+ * @param {IIOSBuildData} iOSBuildData iOS speicific information for the build.
+ * @returns {Promise<void>}
+ */
+validateBuildProperties(platform: string,
+	buildConfiguration: string,
+	projectId: string,
+	androidBuildData?: IAndroidBuildData,
+	iOSBuildData?: IIOSBuildData): Promise<void>;
+```
+Detailed description of each parameter can be found [here](./lib/definitions/cloud-build-service.d.ts).
+</br>
+Usage:
+```JavaScript
+const tns = require("nativescript");
+const fs = require("fs");
+const path = require("path");
+const packageJsonContent = JSON.parse(fs.readFileSync("./package.json", "utf8").toString());
+
+const projectId = packageJsonContent.nativescript.id;
+
+const androidReleaseConfigurationData = {
+	pathToCertificate: "./myCertificate.p12",
+	certificatePassword: "123456"
+};
+
+const platform = "android";
+const buildConfiguration = "release";
+
+tns.cloudBuildService
+	.validateBuildProperties(platform, buildConfiguration, projectId, androidReleaseConfigurationData)
+	.then(buildResult => console.log("Data is valid"))
+	.catch(err => console.error("Data is invalid:", err));
+```
+
 ### Development
 The project is written in TypeScript. After cloning it, you can setup it by executing the following commands in your terminal:
```

Comment:
After cloning it, you can ~setup it~ **set it up** by executing the following commands...